server/npg/porch/endpoints/pipelines.py

```diff
@@ -33,18 +33,18 @@
     "/",
     response_model=List[Pipeline],
     summary="Get information about all pipelines.",
-    description="Get all pipelines. A filter will be applied if used in the query."
+    description="Get all pipelines as a list. A uri filter can be used."
 )
 async def get_pipelines(db_accessor=Depends(get_DbAccessor)) -> List[Pipeline]:
     return await db_accessor.get_all_pipelines()
 
 @router.get(
     "/{pipeline_name}",
-    response_model=List[Pipeline],
+    response_model=Pipeline,
     responses={404: {"description": "Not found"}},
     summary="Get information about one pipeline.",
 )
-async def get_pipeline(pipeline_name: str, db_accessor=Depends(get_DbAccessor)) -> List[Pipeline]:
+async def get_pipeline(pipeline_name: str, db_accessor=Depends(get_DbAccessor)) -> Pipeline:
```

Comment:
If this is turning to singular, I think the variable names like `pipelines` ought to change too.